HW03-Import-and-Graph.Rmd

```diff
@@ -0,0 +1,74 @@
+
+
+## Data import
+First, I imported the data table that I previously uploaded to GitHub:
+
+```{r}
+biomass_URL <- "https://raw.githubusercontent.com/lcespedesarias/HW03/master/CarbonBiomass_byHabitat.csv"
+biomass <- read.csv(url(biomass_URL),h=T,sep=";")
+head(biomass)
+```
+
+## Adjustments in the data table
+Before making the figure I needed to reorder one variable (Habitat) and create a new category called "Habitat type" (which is more general/broad than "Habitat"):
+
+```{r}
+biomass$Habitat <- factor(biomass$Habitat, 
+                          levels = c("Mature_forest","Secondary_Growth",
+                                     "Forest_fragment/Riparian_strip",
+                                     "Shade_coffee","Sun_coffee",
+                                     "Live_Fence/Pasture","Paramo"))
+
+#This will be the new habitat variable
+biomass$Habitat_type[biomass$Habitat %in% c("Mature_forest","Secondary_Growth",
+                                     "Forest_fragment/Riparian_strip")] <- "Forest"
+
+biomass$Habitat_type[biomass$Habitat %in% c("Shade_coffee","Sun_coffee")] <- "Coffee"
+
+biomass$Habitat_type[biomass$Habitat %in% c("Live_Fence/Pasture","Paramo")] <- "Open"
+```
+
+## Figure
+Now the figure. So, what I wanted was a simple figure to show how values of above ground carbon biomass vary by habitat. I also wanted to visually group the specific habitat categories according to the "broad" habitat categories. Therefore, I decided to do a boxplot and color the boxes according to the broad habitat category. This is what I did:
+
+```{r pressure, echo=TRUE}
+ggplot(biomass,mapping = aes(x = Habitat, y = hansen_treeD_1km, fill=Habitat_type)) +
+  #To change the default grey background
+  theme_bw()+
+  #To add the points and boxes (boxes with no outliers)
+  geom_jitter(aes(color=Habitat_type)) +
+  geom_boxplot(alpha=0.4,outlier.colour = NA)+
+  #To add axes labels and adjust their size
+  ylab("Above ground carbon biomass") +
+  xlab("Habitat") +
+  theme(axis.text.x = element_text(color="black", size=8, angle=45,hjust=1))+
+  theme(axis.text.y = element_text(color="black", size=8))+
+  theme(axis.title = element_text(color="black", size=12))+
+  scale_x_discrete(labels=c("Mature forest","Secondary forest","Forest fragment",
+                            "Shade coffee","Sun coffee","Live fences","Paramo"))+
+  #To add title and change the default size
+  ggtitle(label="Above ground forest biomass \n in different habitats")+
+  theme(plot.title = element_text(size = 15))
+
+```
+
+It looks like open habitats (paramo and live fences) do tend to have lower values of above ground carbon biomass than forested habitats (mature forest, secondary growth, forest fragments), as predicted :)
```

Comment:
You know it's a good scientific figure when you can read this statement and immediately understand the point when looking at the graph. Fantastic job!

Reply:
Thanks so much for all the comments!